README.md

```diff
@@ -1,2 +1,2 @@
-# constitution
-The RITSEC club constitution
+# RITSEC Constitution
+The RITSEC club constitution can be downloaded by clicking this link:  https://github.com/ritsec/constitution/releases/download/v1.1/constitution.pdf
```

Comment:
Should be changed to reference https://github.com/ritsec/constitution/releases so that they can just grab the latest once CI is up instead of having to update the readme every time